Python/pywarpx/picmi.py

```diff
@@ -2067,8 +2065,7 @@ def initialize_inputs(self):
             # --- is the same on all processors.
             fields_to_plot = list(fields_to_plot)
             fields_to_plot.sort()
-            self.diagnostic.fields_to_plot = fields_to_plot
-
+            self.diagnostic.set_or_replace_attr('fields_to_plot', fields_to_plot)
```

Comment:
I don't think that this is needed here now since there is only one type of lab frame diagnostic. Though, perhaps the `LabFrameParticleDiagnostic` should be implemented (in a separate PR). Does the BTD in WarpX allow turning on and off the particles and fields?